### PR TITLE
Weather App | Saralie Bognandi & Julia Östedt

### PR DESCRIPTION
**Pull request by juliaostedt**
View it live: https://stockholm-weather.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+// DOM selectors stored as short variables
+const todaysPrompt = document.getElementById("todaysPrompt");
+const header = document.getElementById("header");
+const forecast = document.getElementById("forecast");
+
+// Global variables
```
Comment (miadahlgren) [inline code/script.js:6]: good idea to declare variables this way

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+         <div class="temp"> ${filteredForecast[2].main.temp.toFixed(0)}°</div>
+       </div>
+       <div class="dayTemp">
+         <div class="day">${weekday(filteredForecast[3].dt)}</div>
+         <div class="temp"> ${filteredForecast[3].main.temp.toFixed(0)}°</div>
+       </div>
+       <div class="dayTemp">
+         <div class="day">${weekday(filteredForecast[4].dt)}</div>
+         <div class="temp"> ${filteredForecast[4].main.temp.toFixed(0)}°</div>
+       </div>
```
Comment (miadahlgren) [inline code/script.js:103]: this part is a bit repetitive so a loop would be nice. But we know that you would have added that if you had more time. :) Is there another way to do it?

File: code/style.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+.dayTemp {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+  width: 90%;
+  border-bottom: dashed 1px;
+  text-transform: lowercase;
+}
+
+/* tablet styles */
```
Comment (miadahlgren) [inline code/style.css:39]: the responsiveness works really well! We have nothing bad to say :D

**Review by miadahlgren — COMMENTED** [review]
you did a really good job! if you want to continue working on the project you could change the  const weekday (liness 83-103) into a loop of some sort but apart from that the app looks really great. The code was well structured. //Mia and Antonella

**Review by Sailornina — COMMENTED** [review]
Good Job! :)

**Pull request closed without merging**